apps/parsley/vite.config.ts

```diff
@@ -66,6 +66,7 @@ export default defineConfig({
     }),
     sentryVitePlugin({
       authToken: process.env.PARSLEY_SENTRY_AUTH_TOKEN,
+      disable: process.env.NODE_ENV === "development",
```

Comment:
Importing from `src` causes some TS issues in this file since `vite.config.ts` is excluded from Parsley's `tsconfig.json` and included instead in `tsconfig.node.json`. Since this is a one-off I think using a string is fine.